django/db/backends/postgresql/base.py

```diff
@@ -359,26 +343,50 @@ def get_new_connection(self, conn_params):
             )
         return connection
 
+    @cached_property
```

Comment:
Accessing the settings dict is cheap, do we really need a cached property here?

Reply:
Good point, updated. I had copied `timezone_name` :+1: